src/pages/auth/chat/list/create/modal/ChatRoomModal.tsx

```diff
@@ -8,11 +9,13 @@ type modalProps = {
 };
 
 function ChatRoomModal(props: modalProps) {
-  const [titleInput, setTitleInput] = useState("");
   const [status, setStatus] = useState("");
+  const [titleInput, setTitleInput] = useState("123");
```

Comment:
```suggestion
  const [titleInput, setTitleInput] = useState("");
```

제가 테스트때 항상 쓰는 비번이라 뭔가 하고 놀랐는데 요기 초깃값이 있었네요!